cetmix_tower_server/models/cx_tower_server.py

```
@@ -1193,10 +1220,12 @@ def _command_runner_file_using_template(
         """
         try:
             # Attempt to create a new file using the template for the current server
-            file = log_record.command_id.file_template_id.create_file(
-                server=self,
+            plan_line = log_record.plan_log_id.plan_line_executed_id
+            file = self._command_runner_log_record.command_id.file_using_template_create_file(
```

Ruff (0.12.2)

Line too long (94 > 88)

(E501)